### ISSUE TITLE
Define Hyperparameters and Tuning

### ISSUE THREAD

**Issue by adrian-lison** [adrian-lison/gnn-community-detection#1]
Make list of ANN hyperparameters and their vlaue ranges to scan during the experiments and other tuning approaches such as adaptive learning rates, dropout, batch normalization etc.
